lib/govuk_publishing_components/presenters/meta_tags.rb

```diff
@@ -92,13 +92,12 @@ def add_political_tags(meta_tags)
       def add_ga4_political_tags(meta_tags)
         government = content_item.dig(:links, :government, 0)
 
-        # political: true/false is in a different place to current: true/false, which is why we have 'details' and 'government[:details]'
+        # political: true/false is in a different place to current: true/false, which is why we have 'details' and 'government.dig(:details)'
         if government && details[:political]
-          meta_tags["govuk:ga4-political-status"] = government[:details][:current] ? "political" : "historic"
-
-          government_title = government[:title]
-          if government_title && !government[:details][:current]
-            meta_tags["govuk:ga4-publishing-government"] = government_title
+          current_government = government.dig(:details, :current)
+          unless current_government
```

Comment:
I'm struggling to get my head round this a bit so bear with me. This seems like quite a significant change? It used to always set the `ga4-political-status` meta tag (either to political or historic) but now it seems like it might not set it at all, if there's a `current_government`. And it can only ever be `historic`, not `political`?

Reply:
@andysellick No worries - It no longer gets set to `political` at all with this change since that's only relevant for pages that are published during the current government. Because the PAs have only requested to have the information of previous governments I have removed it from being set. 

Political would get set if `current` is `true` for the government in the content item.  In the old meta tag, it was setting the meta tag to `political` based on that:

```
if details[:political]
  political_status = details[:government][:current] ? "political" : "historic"
  end
```